resources/jvm.options

```diff
@@ -0,0 +1,101 @@
+## JVM configuration
```

Comment:
Give a doc about the original source of this file.

Reply:
OK, I didn't find a sample jvm.options from official sites and took the elastic search sample.

Reply:
OK, I didn't find a sample jvm.options from official sites and took the elastic search sample.